lib/pr_checker.js

```diff
@@ -95,8 +95,12 @@ class PRChecker {
       status = false;
       logger.warn(`Approvals: 0`);
```

Comment:
We can build something like `approvalFromReview` and use that from here, so we don't need to do the filtering in `getTSCHint`.

Reply:
Going to look into this!